### DISCUSSION TITLE
Abbreviations for the UV index categories

### DISCUSSION THREAD

**Discussion by elydpg** [breezy-weather/breezy-weather#1114] [Ideas]
### Describe your idea

Much like wind speed, the UV index has a tendency to get cut off in the header because the text is long, as in this example:

![20240610_104303](https://github.com/breezy-weather/breezy-weather/assets/8622264/73deda99-40af-4505-af12-7994eb5706e2)

However unlike wind speed, this is text that will have a higher chance of getting cut off even on larger phone screens, and thus I think that it's worth having abbreviated strings for the UV index categories in case the full category names cannot fit. For example, in English, we could have

Low -> Lo
Moderate -> Mid
High -> Hi
Very high -> V.hi
Extreme -> Xtrm

Of course the ones for Low and High are kinda pointless but I've included them for completeness, since other languages may need to abbreviate them. Naturally, each language needs its own abbreviations.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.4 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
A new homepage is in progress (https://github.com/breezy-weather/breezy-weather/pull/1928), and the details on top will be removed in favor of Material 3 Expressive blocks.
The new UV block will include the UV index level, and it will not get cut as there will be enough room.
